app/src/main/java/pm/gnosis/heimdall/data/remote/models/push/PushMessages.kt

```diff
@@ -3,23 +3,23 @@ package pm.gnosis.heimdall.data.remote.models.push
 import com.squareup.moshi.Json
 
 sealed class PushMessage(
-    @Json(name = "type") val type: String
```

Comment:
why is the `@Json` annotation removed?

Reply:
`PushMessage` classes are only parsed from maps, therefore I removed all stuff related to moshi